src/posix.mak

```diff
@@ -46,9 +46,9 @@ $(warning ============================== )
 # Forward D compiler bootstrapping to bootstrap.sh
 ifneq (,$(AUTO_BOOTSTRAP))
 default:
-	@./bootstrap.sh
+	@bash ./bootstrap.sh
```

Comment:
This only solves the problem on the Makefile side. Why not use `#!/usr/bin/env bash` on the script itself to read `$PATH` ?

Reply:
Then the problem becomes, where is env?

Reply:
`env` is part of the [POSIX.1](https://www.unix.com/man-page/posix/1posix/env/) and it is implemented on almost every Unix-like system including Linux and BSD-like OSes. Furthermore, this shebang is known to be a portable alternative for `perl` and `python` (which are more likely to be on a different directory due to virtual environments and custom installations).